cylc/flow/suite_status.py

```diff
@@ -27,19 +29,115 @@
 KEY_UPDATE_TIME = "update-time"
 KEY_VERSION = "version"
 
-# Suite status strings.
-SUITE_STATUS_HELD = "held"
-SUITE_STATUS_RUNNING = "running"
-SUITE_STATUS_STOPPING = "stopping"
+# Status message strings
 SUITE_STATUS_RUNNING_TO_STOP = "running to stop at %s"
 SUITE_STATUS_RUNNING_TO_HOLD = "running to hold at %s"
 
-# Pseudo status strings for use by suite monitors.
-#   Use before attempting to determine status:
-SUITE_STATUS_NOT_CONNECTED = "not connected"
-#   Use prior to first status update:
-SUITE_STATUS_CONNECTED = "connected"
-SUITE_STATUS_INITIALISING = "initialising"
-#   Use when the suite is not running:
-SUITE_STATUS_STOPPED = "stopped"
-SUITE_STATUS_STOPPED_WITH = "stopped with '%s'"
+# TODO - the suite status should be a property of the Scheduler instance
+#        rather than derived from scheduler state?
```

Comment:
Good code improvement. And it is indeed weird to have `get_suite_status` here inspecting `Scheduler` and some of its member properties to get the status. Maybe a property in `Scheduler` would work later (but good for now I guess).